llvm/test/CodeGen/X86/huge-stack.ll

```diff
@@ -1,7 +1,19 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
 ; RUN: llc -O0 -mtriple=x86_64 < %s | FileCheck %s --check-prefix=CHECK
 %large = type [4294967295 x i8]
 
 define void @foo() unnamed_addr #0 {
+; CHECK-LABEL: foo:
+; CHECK:       # %bb.0:
+; CHECK-NEXT:    movabsq $8589934462, %rax # imm = 0x1FFFFFF7E
+; CHECK-NEXT:    subq %rax, %rsp
+; CHECK-NEXT:    .cfi_def_cfa_offset 8589934470
+; CHECK-NEXT:    movb $42, {{[0-9]+}}(%rsp)
```

Comment:
When I updated this test to be autogenerated, I overlooked that it replaced the offsets with regex expressions. The offsets are super important for this test so I'll see if `update_llc_test_checks.py` supports not doing this transformation or revert this change and leave them as manual tests.

Reply:
`no_x86_scrub_sp` should do what you're after

Reply:
Yep, that did it. Thanks!